### PR TITLE
add a flag to enable --preview-dart-2

### PR DESCRIPTION
**Pull request by devoncarew**
- add a preference flag to opt into `--preview-dart-2`; fix https://github.com/flutter/flutter-intellij/issues/1667

@pq

### CODE REVIEW

File: src/io/flutter/run/bazel/BazelFields.java [hunk 1]
```diff
@@ -260,7 +261,11 @@ GeneralCommandLine getLaunchCommand(@NotNull Project project,
     commandLine.addParameter("--");
 
     // Tell the flutter tommand-line tools that we want a machine interface on stdio.
```
Comment (scheglov) [inline src/io/flutter/run/bazel/BazelFields.java]: tommand/command

Comment (devoncarew) [reply src/io/flutter/run/bazel/BazelFields.java]: done

**Comment by devoncarew** [comment]
I'm looking into the travis failures (looks like an issue with pub w/ the dart sdk that flutter is using).

**Comment by devoncarew** [comment]
fixed upstream

**Pull request merged**